spec/feature_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'takeaway'
+
+RSpec.describe Takeaway do
+  describe "Feature Tests" do
+    before(:each) do
+      @t = Takeaway.new
```

Comment:
You should use `subject(:takeaway) { described_class.new }`. This way you can have an explicit subject. Also, in general, if you're making variables that are going to be used repeatedly, use `let`, as RSpec knows how to deal with these variables efficiently. In large applications, there can be 100s of tests so it can be expensive to refresh them before each test.